### ISSUE TITLE
Syntax Error report (2021)

### ISSUE THREAD

**Issue by Mafiaboy2020** [android/ndk#1438]
There is an intentional error on the part of Google technologies n android-ndk r22 damn who is programming it, but despite the wasted time of me in order to build a hello world program in a minute of time ...
The errors are as follows:
Nasty file 1:
           setup-toolchain.mk 
           clang++ Syntax Error Unterminated quoted string "
Nasty file 2:
           sanitizers.mk 
           Error msg: python Syntax error "(" unexpected ?!

The fuking ndk and.

**Comment by enh-google** [comment]
you'll need to provide a way for us to actually reproduce this if you want us to look at it...

(building the HelloJNI sample in Android Studio works fine with NDK r22, for example.)

**Comment by Mafiaboy2020** [comment]
Yes, I can direct Google engineers to build an effective compiler
Also, you do not know that there are those who do not own a supercomputer to install Android Stdio
as you know that the Android system is based on the Linux kernel, so why not provide a virtual machine and a good translator, and developers can translate their projects quickly, efficiently and without?  Where projects can be compiled in any work environment ?
